### PR TITLE
added my profile

### PR DESCRIPTION
**Pull request by amanchauhann**

### CODE REVIEW

File: public/data/amanchauhann.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started web development",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "trying to break into tech",
```
Comment (kumarsonsoff3) [inline public/data/amanchauhann.json:24]: What do you think about capitalizing these! 
```suggestion
      "description": "Trying to break into Tech",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Unfortunately you have committed into `package-lock.json`. 

Please Revert the changes from the `package-lock.json` file. Let us know if you need any help!

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Please revert changes to package-lock file then this should be ready to merge.

**Comment by EmmaDawsonDev** [comment]
Do you need any help with this @amanchauhann ?

**Review by Kartik2220 — APPROVED** [review]
code looks good 👍.

**Comment by kumarsonsoff3** [comment]
It's more than a month @amanchauhann, since you responded. So we are closing this PR! 
Feel free to **re-poen** this and make changes or create a new PR containing only your file! Let us know if your need any help with this! 🙂

**Pull request closed without merging**